ebos/eclmpiserializer.hh

```diff
@@ -464,6 +466,22 @@ protected:
         constexpr static bool value = true;
     };
 
+    //! \brief Predicate for maps
+    template<class T>
+    struct is_map {
+        constexpr static bool value = false;
+    };
+
+    template<class T1, class T2>
+    struct is_map<std::map<T1,T2>> {
+        constexpr static bool value = true;
+    };
+
+    template<class T1, class T2>
+    struct is_map<std::unordered_map<T1,T2>> {
+        constexpr static bool value = true;
+    };
```

Comment:
Do we gain anything from restricting these types to their default allocators, comparison predicates, and hash functions?

Reply:
not really, just wanted to keep it simple but i can extend.